src/index.js

```diff
@@ -415,7 +415,7 @@ Multiaddr.protocols = protocols
  */
 Multiaddr.isMultiaddr = function isMultiaddr (addr) {
   if (addr.constructor && addr.constructor.name) {
-    return addr.constructor.name === 'Multiaddr'
+    return (addr instanceof Multiaddr)
```

Comment:
@vmx @dryajov I see that you haven't released yet, good. Can you revert this please?

We never use instanceof anywhere for module classes because it breaks when two versions are being used and func A passes the wrong version to func B and then the check fails.